packages/design-system/src/components/Table/Table.test.tsx

```diff
@@ -1,149 +1,70 @@
-import { mount, shallow } from 'enzyme';
 import React from 'react';
+import { render, screen } from '@testing-library/react';
+
 import Table from './Table';
 import TableCaption from './TableCaption';
+// import TableBody from './TableBody';
 
-const defaultCaptionChildren = 'Foo';
-const tableCaption = <TableCaption>{defaultCaptionChildren}</TableCaption>;
-
-function render(customProps = {}, children = tableCaption, deep = false) {
+const makeTable = (customProps = {}) => {
   const props = Object.assign({}, customProps);
```

Comment:
This line doesn't seem necessary. Could just spread the props directly on line 12

Reply:
Good call.